doc/source/skips/1-governance.rst

```diff
@@ -0,0 +1,153 @@
+.. _governance:
+
+:Author: Juan Nunez-Iglesias <juan.nunez-iglesias@monash.edu>
+:Author: Stéfan van der Walt <stefanv@berkeley.edu>
+:Author: Josh Warner
+:Author: François Boulogne
+:Author: Emmanuelle Gouillart
+:Author: Mark Harfouche
+:Author: Lars Grüter
+:Author: Egor Panfilov
+:Status: Draft
+:Type: Process
+:Created: 2019-07-02
+:Resolved:
+:Resolution:
+:Version effective: 0.16
```

Comment:
This tag is not described in SKIP-0.

Reply:
Indeed, instead `skimage-Version` is described. I kinda like this better. What do you think? Update SKIP-0 or this tag?

Reply:
I have a preference for `skimage-version`, since it clearly shows that we are talking about the software.  I also think it's probably slightly safer to not have spaces in the tags, in case we want to change to a different type of metadata later on.